includes/functions/functions_email.php

```diff
@@ -342,19 +346,19 @@ function zen_mail($to_name, $to_address, $email_subject, $email_text, $from_emai
 
             $mail->Subject = $email_subject;
 
-            if ($email_transport == 'sendmail-f' || EMAIL_SEND_MUST_BE_STORE == 'Yes') {
+            if ($email_transport === 'sendmail-f' || EMAIL_SEND_MUST_BE_STORE === 'Yes') {
                 $mail->Sender = EMAIL_FROM;
             }
 
             // set the reply-to address.  If none set yet, then use Store's default email name/address.
             // If sending from checkout or contact-us, use the supplied info
-            $email_reply_to_address = (isset($email_reply_to_address) && $email_reply_to_address != '') ? $email_reply_to_address : (in_array($module, ['contact_us', 'ask_a_question', 'checkout_extra']) ? $from_email_address : EMAIL_FROM);
-            $email_reply_to_name = (isset($email_reply_to_name) && $email_reply_to_name != '') ? $email_reply_to_name : (in_array($module, ['contact_us', 'ask_a_question', 'checkout_extra']) ? $from_email_name : STORE_NAME);
+            $email_reply_to_address = (isset($email_reply_to_address) && $email_reply_to_address !== '') ? $email_reply_to_address : (in_array($module, ['contact_us', 'ask_a_question', 'checkout_extra']) ? $from_email_address : EMAIL_FROM);
```

Comment:
can't this line and the line after be simplified to:
```
$email_reply_to_address = $email_reply_to_address ?? (in_array($module, ['contact_us', 'ask_a_question', 'checkout_extra']) ? $from_email_address : EMAIL_FROM);
$email_reply_to_name = $email_reply_to_name ?? (in_array($module, ['contact_us', 'ask_a_question', 'checkout_extra']) ? $from_email_name : STORE_NAME);
```

Reply:
I wish.  Unfortunately the `??` operator looks for a value to be `null` and the two values could be an empty string.